docs/sources/upgrade-guide/upgrade-v9.3/index.md

```diff
@@ -0,0 +1,21 @@
+---
+aliases:
+  - ../installation/upgrading/
```

Comment:
I think the second redirect is necessary to redirect the now removed page `setup-grafana/upgrade-grafana/`.

```suggestion
  - ../installation/upgrading/
  - ../setup-grafana/upgrade-grafana/
```